src/main/java/com/scalyr/integrations/kafka/AddEventsClient.java

```diff
@@ -455,12 +455,10 @@ private void writeLogArrayEntry(Map.Entry<Event, Integer> logEntry, JsonGenerato
       jsonGenerator.writeStringField("id", logId.toString());
       jsonGenerator.writeObjectFieldStart("attrs");
 
+      // __origServerHost is a stream identifier that is used to create a virtual session
+      // __origServerHost event attr will be promoted to serverHost session attr
```

Comment:
I would say `__origServerHost event attr will be promoted to serverHost session attr on the server`.